reactive_graph/src/computed/selector.rs

```diff
@@ -55,6 +55,7 @@ use std::{
 ///
 /// # any_spawner::Executor::tick().await;
 /// assert_eq!(is_selected.selected(5), false);
+/// # // TODO: this test is really failing, put `# }).await;` in the next line
```

Comment:
The call to `is_selected.v.read()` returns `Ok(None)`, so it panics with "called `Option::unwrap()` on a `None` value" at `selected()` method.